pom.xml

```diff
@@ -1297,6 +1297,7 @@
                   <exclude>**/*.svg</exclude>
                   <exclude>**/bower_components/**</exclude>
                   <exclude>**/node_modules/**</exclude>
+                  <exclude>**/node/**</exclude>
```

Comment:
shouldn't be at top level pom. Better have it localized in the cdap-ui module (in fact not just `**/node/**`, but also the bower and node_modules one).